lms/templates/wiki/history.html

```diff
@@ -228,9 +228,9 @@ <h2 id="preview-title">{% trans "Wiki Revision Preview" %}<span class="sr">, {%
       {% endif %}
       </div>
     </div>
-  </section>
+  </div>
 
-  <section id="mergeModal" class="modal" aria-hidden="true">
+  <div id="mergeModal" class="modal" aria-hidden="true">
```

Comment:
This might just be the way the dif is being shown, but I do not see a closing div tag.  Is one present in the changes for this line?